monGARS/core/monitor.py

```
@@ -1,10 +1,13 @@
-import psutil
-import GPUtil
 import asyncio
-from dataclasses import dataclass
 from concurrent.futures import ThreadPoolExecutor
+from dataclasses import dataclass
+
+import GPUtil
+import psutil
+
 from monGARS.config import get_settings
```

Ruff (0.12.2)

`monGARS.config.get_settings` imported but unused

Remove unused import: `monGARS.config.get_settings`

(F401)